heron/api/src/java/org/apache/heron/api/Config.java

```diff
@@ -215,6 +215,11 @@
    * spout0:12434,spout1:345353,bolt1:545356.
    */
   public static final String TOPOLOGY_COMPONENT_DISKMAP = "topology.component.diskmap";
+  /**
+   * Sorting strategy for FirstFitDecreasingPacking algorithm.
+   * RAM_FIRST (default), or CPU_FIRST
+   */
+  public static final String TOPOLOGY_FFD_SORTING_STRATEGY = "topology.ffd.sorting.strategy";
```

Comment:
I am wondering if there is a better place for this config. @huijunw @nlu90 any thoughts?

Also if it will be kept here. I would suggest make the name and string like topology.packing.ffd....